test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -710,7 +710,7 @@ export default function Main(props: WalletConnectionProps) {
                                                 JSON.stringify('number')
                                             ) {
                                                 // eslint-disable-next-line prefer-destructuring
-                                                attributes[attributeSchema[Number(key)][0]] = Number(
+                                                attributes[attributeSchema[Number(key)][0]] = BigInt(
```

Comment:
I tested it with a `number` attribute in the schema:

e.g. https://gist.githubusercontent.com/DOBEN/15f6e2ba8f4c99f134264a14327a8017/raw/e2315882cc490904bee596242a8f122054f2565b/gistfile1.txt


Getting this error in the developer console:
![Screenshot from 2023-08-21 18-58-39](https://github.com/Concordium/concordium-web3id/assets/13039169/50e01bb7-3ac8-479e-8657-cd0143d752f4)